core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -114,6 +115,30 @@ LoggingProperties loggingProperties() {
 
       /* Labels to identify jobs managed by this job coordinator */
       private Map<String, String> jobSelector = new HashMap<>();
+
+      /* Selectors to define featureSets that are responsibility of current JobManager */
+      private List<Selector> featureSetSelector = new ArrayList<>();
+
+      /* Specify names of stores that must not be used by current JobManager */
+      private List<String> blacklistedStores = new ArrayList<>();
```

Comment:
Is it easier to go the reverse? stores that are whitelisted?

I can imagine that a JobCoordinator will be deployed only for a list of stores, and they will probably be hardcoded as part of deployment.

Reply:
Use case I had in my mind was that we don't want to have two job coordinators writing to bigquery. So we turning off just one this store.
In reverse case we need to specify all stores names that are distributed across all serving configs.

Reply:
But right now it's not a lot of them. So I guess both options would work

Reply:
> Use case I had in my mind was that we don't want to have two job coordinators writing to bigquery. So we turning off just one this store.

Are you sure there isnt a use case like this? I think it might theoretically be useful during rollouts, even though its not a part of our plan now. There is also no downside to having duplicated rows if they are deduplicated at query time.

> In reverse case we need to specify all stores names that are distributed across all serving configs.

Yea, but understanding which stores to exclude is a lot harder. I know my store, but I dont want to keep track of other people's stores. So lets say we have 5-6 job coordinators across multiple VPC. They each have 1 store. Does this mean that they need to exclude all stores except their own?